src/ControlSurface/Controls/Slider/ScoreSliders.qml

```diff
@@ -61,39 +69,38 @@ ColumnLayout {
                 var tmpFrom, tmpValue, tmpTo
                 var tmpStepSize
                 switch (s.uuid) {
-                    // Float Slider
                 case "af2b4fc3-aecb-4c15-a5aa-1c573a239925":
+                    // Float Slider
                     tmpFrom = s.Domain.Float.Min
                     tmpValue = s.Value.Float
                     tmpTo = s.Domain.Float.Max
                     tmpStepSize = 0.0
                     break
-                    // Int Slider
                 case "348b80a4-45dc-4f70-8f5f-6546c85089a2":
+                    // Int Slider
                     tmpFrom = s.Domain.Int.Min
                     tmpValue = s.Value.Int
                     tmpTo = s.Domain.Int.Max
                     tmpStepSize = 1
                     break
                 }
                 sliderListModel.insert(sliderListModel.count, {
-                                           "myName": s.Custom,
-                                           "myId": s.id,
-                                           "myFrom": JSON.stringify(tmpFrom),
-                                           "myValue": tmpValue,
-                                           "myTo": JSON.stringify(tmpTo),
-                                           "myStepSize": tmpStepSize,
-                                           "myInd": ind,
-                                           "myUuid": s.uuid
+                                           "_custom": s.Custom,
```

Comment:
why the _ ? did it break because of the "id" property ?

Reply:
I have not tested it but I have assumed it